src/openbench/metric/keyword_boosting_metrics/boosting_metrics.py

```diff
@@ -0,0 +1,265 @@
+                    phrase_ref = " ".join([item[0] for item in item_ref])
+                    phrase_hyp = " ".join([ali[item[1]][1] for item in item_ref])
+                    if phrase_ref in key_words_stat:
+                        key_words_stat[phrase_ref][1] += 1  # add to gt
+                        if phrase_ref == phrase_hyp:
+                            key_words_stat[phrase_ref][0] += 1  # add to tp
+
+            # For false positive hypothesis phrase
+            idx = 0
+            item_hyp = []
+            while idx < len(ali):
+                if item_hyp:
+                    item_hyp = [item_hyp[1]]
+                    idx = item_hyp[0][1] + 1
+                while len(item_hyp) != ngram_order and idx < len(ali):
+                    word = ali[idx][1]
+                    idx += 1
+                    if word == eps:
+                        continue
+                    else:
+                        item_hyp.append((word, idx - 1))
+                if len(item_hyp) == ngram_order:
+                    phrase_hyp = " ".join([item[0] for item in item_hyp])
+                    phrase_ref = " ".join([ali[item[1]][0] for item in item_hyp])
+                    if phrase_hyp in key_words_stat and phrase_hyp != phrase_ref:
+                        key_words_stat[phrase_hyp][2] += 1  # add to fp
+
+        # Compute totals
+        tp = sum([key_words_stat[x][0] for x in key_words_stat])
+        gt = sum([key_words_stat[x][1] for x in key_words_stat])
+        fp = sum([key_words_stat[x][2] for x in key_words_stat])
+
+        print(f"TP: {tp}, FP: {fp}, FN: {gt - tp}")
+        
+        # Print detailed keyword statistics
+        fp_keywords = []
+        for i, keyword in enumerate(normalized_keywords, 1):
+            if keyword in key_words_stat:
+                kw_tp = key_words_stat[keyword][0]
+                kw_gt = key_words_stat[keyword][1]
+                kw_fp = key_words_stat[keyword][2]
+                
+                if kw_gt > 0:  # Only print for keywords that exist in ground truth
+                    if kw_tp > 0:
+                        status = f"TP ({kw_tp}/{kw_gt})"
+                    else:
+                        status = f"FN (0/{kw_gt})"
+                    print(f"{keyword} {i}/{len(normalized_keywords)} - {status}")
+                
+                # Collect FP keywords
+                if kw_fp > 0:
+                    fp_keywords.append(f"{keyword} ({kw_fp})")
+        
+        # Print FP keywords separately
+        if fp_keywords:
+            print(f"FP keywords: {fp_keywords} - FP rate: {fp}/{len(normalized_keywords)}")
+        
+        print("---")
+
+        return {
+            "true_positives": tp,
+            "ground_truth": gt,
+            "false_positives": fp,
+            "keyword_stats": key_words_stat
+        }
+
+
+@MetricRegistry.register_metric(PipelineType.BOOSTING_TRANSCRIPTION, MetricOptions.KEYWORD_FSCORE)
```

Comment:
Can we rename `BOOSTING_TRANSCRIPTION` to `KEYWORD_BOOSTED_TRANSCRIPTION`? (or similar)